tests/ffi/network/main.fmf

```diff
@@ -0,0 +1,8 @@
+summary: Sending large packets
+test: ./test_largepckts_icmp.sh 'ping -i 0.01 -s 1600 %s'
```

Comment:
Consider one of the two
chmod +x ./test_largepckts_icmp.sh
or use this 
/bin/bash  ./test_largepckts_icmp.sh